setup.py

```diff
@@ -16,7 +16,7 @@
 ]
 
 tests_require = [
-    'elasticsearch>=1.0.0,<2.0.0',
+    'elasticsearch>=2.0.0,<3.0.0',
```

Comment:
We might need to either pick a version for testing or pull the `elasticsearch` dependency out of here and manage in tox / travis so we can treat them interchangeably